app/src/molecules/DeckThumbnail/index.tsx

```
@@ -1,179 +1,258 @@
   THERMOCYCLER_MODULE_V1,
+  inferModuleOrientationFromXCoordinate,
 } from '@opentrons/shared-data'
 import {
-  parseInitialLoadedLabwareBySlot,
-  parseInitialLoadedLabwareByModuleId,
-  parseInitialLoadedModulesBySlot,
-  parseLiquidsInLoadOrder,
-  parseLabwareInfoByLiquidId,
   parseInitialLoadedLabwareByAdapter,
+  parseLabwareInfoByLiquidId,
 } from '@opentrons/api-client'
-import { getWellFillFromLabwareId } from '../../organisms/Devices/ProtocolRun/SetupLiquids/utils'
-import { getIsOnDevice } from '../../redux/config'
+
 import { getStandardDeckViewLayerBlockList } from './utils/getStandardDeckViewLayerBlockList'
-import { getStandardDeckViewBox } from './utils/getStandardViewBox'
+import { getDeckConfigFromProtocolCommands } from '../../resources/deck_configuration/utils'
+import { getLabwareRenderInfo } from '../../organisms/Devices/ProtocolRun/utils/getLabwareRenderInfo'
+import { getProtocolModulesInfo } from '../../organisms/Devices/ProtocolRun/utils/getProtocolModulesInfo'
+import { useAttachedModules } from '../../organisms/Devices/hooks'
+import { getAttachedProtocolModuleMatches } from '../../organisms/ProtocolSetupModulesAndDeck/utils'
+import { LabwareInfoOverlay } from '../../organisms/Devices/ProtocolRun/LabwareInfoOverlay'
+import { getWellFillFromLabwareId } from '../../organisms/Devices/ProtocolRun/SetupLiquids/utils'
 
 import type { StyleProps } from '@opentrons/components'
 import type {
-  DeckSlot,
-  Liquid,
-  LoadedLabware,
-  RunTimeCommand,
+  CompletedProtocolAnalysis,
+  ProtocolAnalysisOutput,
 } from '@opentrons/shared-data'
 
 interface DeckThumbnailProps extends StyleProps {
-  commands: RunTimeCommand[]
-  labware: LoadedLabware[]
-  liquids?: Liquid[]
-  showSlotLabels?: boolean
+  protocolAnalysis: CompletedProtocolAnalysis | ProtocolAnalysisOutput | null
 }
 
-export function DeckThumbnail(props: DeckThumbnailProps): JSX.Element {
-  const {
-    commands,
-    liquids,
-    labware = [],
-    showSlotLabels = false,
```

we want leave this prop in, and defaulted to false to hide the `SlotLabels`. only the ODD protocol details map version of `DeckThumbnail` shows slot labels.